resharper/resharper-unity/src/HlslSupport/UnityHlslCppCompilationPropertiesProvider.cs

```diff
@@ -28,6 +29,10 @@ public UnityHlslCppCompilationPropertiesProvider(UnityVersion unityVersion)
                 if (!path.IsEmpty)
                     properties.IncludePaths.Add(path);
 
+                properties.PredefinedMacros.Add(CppPPDefineSymbol.ParsePredefinedMacro("SHADER_API_D3D11"));
+                properties.IncludePaths.Add(globalCache.Solution.SolutionDirectory);
+                properties.IncludePaths.Add(globalCache.Solution.SolutionDirectory.Combine("Library"));
```

Comment:
Same comment as `ShaderLabInclusionContextProvider` - do we have shaders that are relative to `Library`?

Reply:
Same answer :-)